docs/internals/contributing/index.txt

```diff
@@ -24,6 +24,10 @@ contribute in many ways:
   ecosystem of pluggable applications is a big strength of Django, help us
   build it!
 
+* Join the `Django forum`_. This is a place for discussing the Django framework
+  and applications and projects that use it. This will be a good place to ask
+  questions or join in the discussion about contributing to the Django project.
```

Comment:
I would move this point below the _"#django IRC channel"_ and make a small rewording:
```
* Join the `Django forum`_. This forum is a place for discussing the Django
  framework and applications and projects that use it. This is also a good
  place to ask and answer any questions related to installing, using, or
  contributing to Django.
```

Reply:
What about top of the list, above mailing list and IRC? (@felixxm You say!)

👍 on @felixxm's suggested wording.

Reply:
Agreed, it can go to the top of the list :1st_place_medal:

Reply:
Perfect, will make the required adjustments and update this PR. Thanks all!